src/hooks/requests/useUsers.ts

```diff
@@ -198,3 +198,44 @@ export const useDeactivateUser = ({ userId }: { userId?: string }) => {
     isLoading,
   }
 }
+
+export const useActivateUser = ({ userId }: { userId?: string }) => {
```

Comment:
**[blocker]**

Didn't notice before, but this also applies for useDeactivateUser:

Currently using this hook means that we need to pass the userId in 2 different places, which can be really errorprone + makes this less convenient to use. Since we need the userId in the constructor of the hook, we can remove it from the "mutationFn" arguments